os/surface_format.h

```diff
@@ -8,12 +9,20 @@
 #define OS_SURFACE_FORMAT_H_INCLUDED
 #pragma once
 
+#include <cstdint>
+
 namespace os {
 
   enum SurfaceFormat {
     kRgbaSurfaceFormat,
   };
 
+  enum PixelAlpha {
```

Comment:
Prefer enum class (we should do the same for SurfaceFormat in a future, it's an old enum):
```suggestion
  enum class PixelAlpha {
```

Reply:
I made a separate commit making SurfaceFormat an enum class.